lametro/templates/lametro/legislation.html

```diff
@@ -97,20 +98,19 @@ <h3 class="no-pad-bottom"><i class='fa fa-fw fa-list-ul'></i> History</h3>
                             </tr>
                         </thead>
                         <tbody>
-                            {% for action in actions %}
+                            {% for action in events %}
                                 <tr>
                                     <td class='nowrap text-muted small'>
-                                        <span datetime='{{action.date_dt | format_date_sort }}'>{{action.date_dt|date:'n/d/y'}}</span>
+                                        <span datetime='{{action.date}}'>{{action.date}}</span>
```

Comment:
Looks like you dropped the formatting for display here.

```suggestion
                                        <span datetime='{{action.date}}'>{{action.date|date:'n/d/y'}}</span>
```